CHANGELOG.md

```diff
@@ -3,6 +3,30 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.23.5
+
+_Dec 27, 2024_
+
+Here are some highlights ✨:
+
+- 🐞 Fix version mismatch issue in codesandbox/stackblitz demos
```

Comment:
Maybe this highlight should be added to Data Grid section? 🤔

Reply:
Did we have section specific highlights ever 🤔 
How about doing this instead?
```suggestion
- 🐞 Fix version mismatch issue in Data Grid codesandbox/stackblitz demos
```